lib/skins.js

```diff
@@ -292,13 +293,35 @@ module.exports = function (gulp, gutil) {
     });
   }
 
+  function checkVersionForThemeSupport() {
+    var server = require('../lib/server.js')(gulp, gutil);
+    var versionCheck = require('../lib/version-check.js')(gulp, gutil);
+    let asyncFn = () => {
```

Comment:
This _let_ and the following ones can be _const_ I think

Reply:
okay, let me change it to const